src/components/Pages/Departments.js

```diff
@@ -10,23 +10,16 @@ import PageHeader from 'components/PageHeader';
 import SectionHeader from 'components/SectionHeader';
 import TileGroup from 'components/Tiles/TileGroup';
 
-const Departments = ({ departments, intl }) => {
-  const departmentLinks = departments.map(d => ({
-    url: `/${d.slug}`,
-    title: d.title,
-  }));
-
```

Comment:
does `departments` now come back in this shape from static.config, which is why theres no need to make `departmentLinks` anymore? (does it even come from static.config?)

Reply:
yeah, just realized my comments weren't submitted on janis... sorry about that.

Reply:
https://github.com/cityofaustin/janis/pull/559/files#r337908132